setup.py

```diff
@@ -22,7 +21,8 @@
     "mock",
     "requests-mock>=1.8.0",
     "httpretty>=1.1.4",
-    "urllib3<2.3.0",  # httpretty doesn't work properly with urllib3>=2.3.0. See #700 and https://github.com/gabrielfalcao/HTTPretty/issues/484
+    "urllib3<2.3.0",
+    # httpretty doesn't work properly with urllib3>=2.3.0. See #700 and https://github.com/gabrielfalcao/HTTPretty/issues/484
```

Comment:
any reason to change this ?

note that this comment is about urllib dependency, now it seems to be about netcdf.